src/database/models/team_item.rs

```diff
@@ -520,4 +520,40 @@ impl TeamMember {
             Ok(None)
         }
     }
+
+    pub async fn get_from_user_id_collection<'a, 'b, E>(
```

Comment:
this should be moved to user_item since collections should be associated with per-user